packages/forms/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+	"extends": "../../.eslintrc",
+	"rules": {
+		"import/no-cycle": 0,
```

Comment:
This is usually bad, but UIForm has a unique Widget entry point, that relies on every widget. Some of them, has nested content (example: fieldset), that render a Widget for each nested element.
That's the cycle.